Add simple drag & lift to the simulation

Nufflee opened the pull request:
This adds drag and lift forces to the simulation based on the coefficients and reference areas.
Closes #10.

Nufflee committed:
Add simple drag & lift to the simulation.

TheGreatFez commented on Simulation/Derivatives.m:
```diff
@@ -1,9 +1,25 @@
+
+  Gravity = [0, -9.81]; % Acceleration due to gravity
+
+  Density = 1.225;                              % Standard air density (kg/m3)
```
Just a FYI, this info will probably be generated from the Atmosphere model in issue #5

TheGreatFez commented on Simulation/Main.m:
```diff
@@ -1,30 +1,36 @@
 % Object parameters
-Plane.Mass = 1;
+Plane.Mass        = 1; % Mass (kg)
+Plane.DragRefArea = 1; % Cross sectional area used for drag (m2)
+Plane.LiftRefArea = 0; % Projected wing area used for lift (m2)
```
The reference area is a single value for the whole aiplane, typically the cross sectional area of the whole plane.

Combine these two variables into one reference area variable.

Nufflee replied:
Are they really the same value? I know that reference area for drag is the cross sectional area of the plane but isn't the reference area for lift the actual area of the wings, because that's what matters for lift?

TheGreatFez replied:
They are the same value. This is needed so that they can be compared equally. The choice for what the reference area will be arbitrary since it only affects the scale of the Cd and Cl but by convention things like "Top down" area or "Front facing" area is used so that the scales across multiple types of aircraft/car/rocket designs are relatively equal.

Nufflee replied:
The change has been made in 06a0894507f717b10d2412fbf40733a744029dae.

TheGreatFez commented on Simulation/Derivatives.m:
```diff
@@ -1,9 +1,25 @@
+  Density = 1.225;                              % Standard air density (kg/m3)
+  Q       = 0.5 * Density * Speed^2;            % Dynamic pressure
+  Drag    = Object.DragRefArea * Object.Cd * Q; % Drag force
+  Lift    = Object.LiftRefArea * Object.Cl * Q; % Lift force
```
See the action in the Main.m regarding the Reference Area, these need to be changed after that is addressed

TheGreatFez reviewed (changes requested):
There are only two changes needed to be made.

In the future, make sure you either:
A) add more discrete commits with short comments
or
B) add more detailed comments to the commits.

This is a relatively small change to the Simulation, but with only a short comment in the commit I can't tell why some things were changed (even if something was done for styling or formatting purposes)

Nufflee committed:
Use the same reference area for drag and lift.

I previously used two separate reference areas for drag and lift calculation but the same area should be used in order to be able to properly compare drag and lift forces.
